resources.md

```diff
@@ -4,3 +4,12 @@ title: Resources
 ---
 
 # A list of x-gov software resources
+---
+
+### Public civil service guidance and standards
+- <a href="https://gds-way.cloudapps.digital/">GDS Way</a>
```

Comment:
I think we can also do this in the markdown format 
[text](url) 
Will deploy and see if all good

Reply:
Yes of course, not sure why I went half and half.

Half asleep! :)

Reply:
Let me know if you'd prefer me to change before merging.